bc/subscription/utils/courtlistener.py

```diff
@@ -308,3 +328,43 @@ def handle_multi_defendant_cases(queue):
         # TODO
         raise NotImplementedError
     logger.debug("handle_multi_defendant_cases(): done")
+
+
+def is_bankruptcy(docket: DocketDict) -> bool:
```

Comment:
We can use just the `court_id` as a key identifier for bankruptcy records. We don't need a list of bankruptcy courts. [Juriscraper](https://github.com/freelawproject/juriscraper) employs the following method to identify bankruptcy dockets:

```python
if self.court_id.endswith("b"):
    self.is_bankruptcy = True
else:
    self.is_bankruptcy = False
```

We can simplify it to:

```python
def is_bankruptcy(docket: DocketDict) -> bool:
    ...
    return docket["court_id"].endswith("b")
```

Reply:
After further code review in this PR, I noticed that the `court_id` is already stored in the **Subscription** model. Therefore, for the `is_bankruptcy` method, we can directly pass the `court_id` and leverage the `endswith` method to identify bankruptcy dockets:

```python
def is_bankruptcy(court_id: str) -> bool:
    ...
    return court_id.endswith("b")
```